src/endpoint/s3/ops/s3_get_bucket_notification.js

```diff
@@ -16,23 +16,28 @@ async function get_bucket_notification(req) {
             conf.Event = conf.event;
             conf.Topic = conf.topic;
             conf.Id = conf.id;
-            delete conf.vent;
+            delete conf.event;
             delete conf.topic;
             delete conf.id;
         }
     }
 
+    const TopicConfiguration = [];
+    if (result && result.length > 0) {
+        for (const conf of result) {
+            TopicConfiguration.push({TopicConfiguration: conf});
```

Comment:
The only issue I could see is that this line 28 could have been placed after line 21 and that could have worked. off course line 25 should be moved up also. Else  things looks good to me.

Reply:
Fixed.